ShanoirChallengeMigrator/.project

```diff
@@ -20,4 +20,15 @@
 		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 	</natures>
+	<filteredResources>

```

Comment:
Can you explain ?

Reply:
This was probably generated by eclipse or another IDE, I just removed it.

Reply:
it says "__CREATED_BY_JAVA_LANGUAGE_SERVER__"